package.json

```diff
@@ -90,122 +91,144 @@
                 "key": "shift+alt+up",
-                "when": "editorTextFocus && !editorReadonly && editorLangId == markdown && !suggestWidgetVisible"
+                "when": "editorTextFocus && !editorReadonly  && !suggestWidgetVisible",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
             },
             {
                 "command": "markdown.extension.onCopyLineDown",
                 "key": "shift+alt+down",
-                "when": "editorTextFocus && !editorReadonly && editorLangId == markdown && !suggestWidgetVisible"
+                "when": "editorTextFocus && !editorReadonly  && !suggestWidgetVisible",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
             },
             {
                 "command": "markdown.extension.onIndentLines",
                 "key": "ctrl+]",
                 "mac": "cmd+]",
-                "when": "editorTextFocus && editorLangId == markdown && !suggestWidgetVisible"
+                "when": "editorTextFocus  && !suggestWidgetVisible",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
             },
             {
                 "command": "markdown.extension.onOutdentLines",
                 "key": "ctrl+[",
                 "mac": "cmd+[",
-                "when": "editorTextFocus && editorLangId == markdown && !suggestWidgetVisible"
+                "when": "editorTextFocus  && !suggestWidgetVisible",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
             },
             {
                 "command": "markdown.extension.checkTaskList",
                 "key": "alt+c",
-                "when": "editorTextFocus && editorLangId == markdown"
+                "when": "editorTextFocus",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
             },
             {
                 "command": "markdown.extension.togglePreview",
                 "key": "ctrl+shift+v",
                 "mac": "cmd+shift+v",
-                "when": "!terminalFocus"
+                "when": "!terminalFocus",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
             },
             {
                 "command": "markdown.extension.togglePreviewToSide",
                 "key": "ctrl+k v",
                 "mac": "cmd+k v",
-                "when": "!terminalFocus"
+                "when": "!terminalFocus",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
             },
             {
                 "command": "markdown.extension.editing.paste",
                 "key": "ctrl+v",
                 "mac": "cmd+v",
-                "when": "editorTextFocus && editorLangId == markdown && editorHasSelection"
+                "when": "editorTextFocus  && editorHasSelection",
+                "injectTo": ["text.html.markdown.jsx", "text.html.markdown"]
```

Comment:
`injectTo` does not exist in `keybindings`:

* <https://code.visualstudio.com/api/references/contribution-points#contributes.keybindings>
* <https://github.com/microsoft/vscode/blob/de68da88f9cf36a2e3d521bbaa3eb4654c1b2c52/src/vs/platform/extensions/common/extensions.ts#L46-L53>
* Theoretically, injecting a key binding into a TM scope doesn't make sense.

Should revert.